packages/fhir-converter/src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -660,8 +660,11 @@ module.exports.external = [
         var ret = {};
   
         for (var t = 0; t < templateIds.length - 1; t++) { // -1 because templateIds includes the full message at the end
-          for (var i = 0; i < msg.ClinicalDocument.component.structuredBody.component.length; i++) {
-            let sectionObj = msg.ClinicalDocument.component.structuredBody.component[i].section;
+          let components = Array.isArray(msg.ClinicalDocument.component.structuredBody.component) 
```

Comment:
bug in CDA to FHIR that would cause it to not parse anything if one component in structured b ody